charts/presidio/Chart.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
 description: A context aware, born to the cloud, customizable data loss prevention service
 name: presidio
-version: unstable
-appVersion: unstable
```

Comment:
you can't remove appVersion

Reply:
i can it is optional. https://docs.helm.sh/developing_charts/

Reply:
No you can't. The chart depends on it
 https://github.com/Microsoft/presidio/blob/development/charts/presidio/templates/analyzer-deployment.yaml#L23

Reply:
got it. I suggest returning appVersion and adding 'latest' as default instead of unstable. as now we have latest version. this change will make it consistent with the helm defaults...